Description: Disable dask theme
Author: Santiago Vila <sanvila@debian.org>
Last-Update: 2024-11-12

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -130,7 +130,7 @@
 
 # -- Options for HTML output ----------------------------------------------
 
-html_theme = "dask_sphinx_theme"
+# html_theme = "dask_sphinx_theme"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
