Description: Fix sphinx configuration
Forwarded: https://github.com/dask/distributed/pull/8434
Author: Julian Gilbey <jdg@debian.org>
Last-Update: 2023-12-31

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -385,8 +385,8 @@
 # Link to GitHub issues and pull requests using :pr:`1234` and :issue:`1234`
 # syntax
 extlinks = {
-    "issue": ("https://github.com/dask/distributed/issues/%s", "GH#"),
-    "pr": ("https://github.com/dask/distributed/pull/%s", "GH#"),
+    "issue": ("https://github.com/dask/distributed/issues/%s", "GH#%s"),
+    "pr": ("https://github.com/dask/distributed/pull/%s", "GH#%s"),
 }
 
 # Configuration for intersphinx: refer to the Python standard library
