From: Debian Python Team <team+python@tracker.debian.org>
Date: Sat, 26 Dec 2020 16:55:39 -0800
Subject: override-worker-test
Forwarded: not-needed

---
 distributed/tests/test_worker.py | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

--- a/distributed/tests/test_worker.py
+++ b/distributed/tests/test_worker.py
@@ -248,7 +248,10 @@
 @gen_cluster(client=True)
 async def test_upload_egg(c, s, a, b):
     eggname = "testegg-1.0.0-py3.4.egg"
-    local_file = __file__.replace("test_worker.py", eggname)
+    if 'AUTOPKGTEST_TMP' in os.environ:
+        local_file = os.path.abspath(eggname)
+    else:
+        local_file = __file__.replace('test_worker.py', eggname)
     assert not os.path.exists(os.path.join(a.local_directory, eggname))
     assert not os.path.exists(os.path.join(b.local_directory, eggname))
     assert a.local_directory != b.local_directory
@@ -277,7 +280,10 @@
 @gen_cluster(client=True)
 async def test_upload_pyz(c, s, a, b):
     pyzname = "mytest.pyz"
-    local_file = __file__.replace("test_worker.py", pyzname)
+    if 'AUTOPKGTEST_TMP' in os.environ:
+        local_file = os.path.abspath(pyzname)
+    else:
+        local_file = __file__.replace('test_worker.py', pyzname)
     assert not os.path.exists(os.path.join(a.local_directory, pyzname))
     assert not os.path.exists(os.path.join(b.local_directory, pyzname))
     assert a.local_directory != b.local_directory
